mmir-plugin-asr-bingWS-web/www/webasrBingWsImpl.js

```diff
@@ -0,0 +1,680 @@
+﻿/*
+ * 	Copyright (C) 2012-2016 DFKI GmbH
+ * 	Deutsches Forschungszentrum fuer Kuenstliche Intelligenz
+ * 	German Research Center for Artificial Intelligence
+ * 	http://www.dfki.de
+ * 
+ * 	Permission is hereby granted, free of charge, to any person obtaining a 
+ * 	copy of this software and associated documentation files (the 
+ * 	"Software"), to deal in the Software without restriction, including 
+ * 	without limitation the rights to use, copy, modify, merge, publish, 
+ * 	distribute, sublicense, and/or sell copies of the Software, and to 
+ * 	permit persons to whom the Software is furnished to do so, subject to 
+ * 	the following conditions:
+ * 
+ * 	The above copyright notice and this permission notice shall be included 
+ * 	in all copies or substantial portions of the Software.
+ * 
+ * 	THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS 
+ * 	OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF 
+ * 	MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
+ * 	IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY 
+ * 	CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+ * 	TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE 
+ * 	SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * Media Module: Implementation for Speech Recognition via the Google Web Speech Recognition service v1 using a proxy/mediator server
+ * 
+ * 
+ * @requires Cross-Domain access, if proxy/mediator server is not located in the same domain as the web app
+ * @requires CSP for accessing the proxy/mediator server via ws: or wss:, e.g. "connect-src ws://server-address" or "default-src connect-src ws://server-address"
+ * 
+ */
+newWebAudioAsrImpl = (function BingWsWebAudioInputImpl() {
+	
+
+	/**  @memberOf BingWsWebAudioInputImpl# */
+	var MODE = 'bingWs';
+
+	/**  @memberOf BingWsWebAudioInputImpl# */
+	var _pluginName = 'bingWsWebAudioInput';
+
+	/** 
+	 * @type mmir.LanguageManager
+	 * @memberOf BingWsWebAudioInputImpl#
+	 */
+	var languageManager = require('languageManager');

```

Comment:
please update use of `require()` for new API version

see [impl. of mmir-plugin-asr-nuance-web](mmir-plugin-asr-nuance-web/www/webasrNuanceImpl.js) for an example:  
line [44](https://github.com/mmig/mmir-plugins-media/blob/master/mmir-plugin-asr-nuance-web/www/webasrNuanceImpl.js#L44) to [77](https://github.com/mmig/mmir-plugins-media/blob/master/mmir-plugin-asr-nuance-web/www/webasrNuanceImpl.js#L77) is an impl. of a require-helper function that supports old and new API versions of the mmir-lib